src/Model/CustomerInterface.php

```diff
@@ -246,4 +240,9 @@ public function saveWithOptions(SaveOptions $saveOptions, $disableVersions = fal
      * @return CustomerSaveManagerInterface
      */
     public function getSaveManager();
+
+    /**
+     * @return string
+     */
+    public function __toString();
```

Comment:
ElementInterface have __toString since Pimcore X. I tried to add it via stub, but this doesn't work. I add a TODO comment so we can removed it when we drop Pimcore 6.9